zebra-chain/src/transaction/sighash.rs

```diff
@@ -24,10 +24,15 @@ const ZCASH_SHIELDED_SPENDS_HASH_PERSONALIZATION: &[u8; 16] = b"ZcashSSpendsHash
 const ZCASH_SHIELDED_OUTPUTS_HASH_PERSONALIZATION: &[u8; 16] = b"ZcashSOutputHash";
 
 bitflags::bitflags! {
+    /// The different SigHash types, as defined in https://zips.z.cash/zip-0143
     pub struct HashType: u32 {
+        ///
         const ALL = 0b0000_0001;
+        ///
         const NONE = 0b0000_0010;
+        ///
         const SINGLE = Self::ALL.bits | Self::NONE.bits;
+        ///
```

Comment:
The empty doc comments prevent a linter from ever noticing that they're empty, so I think it would be preferable to avoid adding them.